.circleci/config.yml

```diff
@@ -2,8 +2,9 @@
 version: 2.1
 
 orbs:
-  coverage-reporter: codacy/coverage-reporter@13.13.7
+  coverage-reporter: codacy/coverage-reporter@13.16.5
   codecov: codecov/codecov@3.2.5
+  skip: true
```

Comment:
```suggestion
```
We already have `skip: true`: https://github.com/ESMValGroup/ESMValCore/blob/188ee5348ffc485c909c2960be0e5fa44f75c6da/.circleci/config.yml#L137